fiona/ogrext.pyx

```diff
@@ -1207,6 +1278,80 @@ cdef class WritingSession(Session):
         gdal_flush_cache(cogr_ds)
         log.debug("Flushed data source cache")
 
+    def set_tags(self, tags, ns=None):
```

Comment:
@snowman2 what what you think about renaming this to update_tags to align with https://rasterio.readthedocs.io/en/latest/api/rasterio._io.html?highlight=set_tags#rasterio._io.DatasetWriterBase.update_tags?

Reply:
Sounds like a good idea to me - just pushed some changes.